core/src/main/resources/hudson/logging/LogRecorderManager/new.jelly

```diff
@@ -27,17 +27,17 @@ THE SOFTWARE.
 -->
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:l="/lib/layout" xmlns:f="/lib/form">
-  <l:layout>
+  <l:layout title="${%New log recorder}">
     <st:include page="sidepanel.jelly" />
 
+    <l:app-bar title="${%New log recorder}"/>
     <l:main-panel>
-      <f:form method="post" action="newLogRecorder">
+      <f:form name="configSubmit" method="post" action="newLogRecorder">
         <f:entry title="${%Name}">
-          <input type="text" name="name" class="jenkins-input" id="box" autofocus="autofocus" />
+          <f:textbox name="name" checkUrl="'${rootURL}/log/checkNewName?name='+encodeURIComponent(this.value)" />
```

Comment:
added validation to match what's done on submission, error message is not very nice otherwise.

still not great that you can submit known bad input, but that's a very separate piece of work